maintainer/deploy_docs.sh

```diff
@@ -44,12 +44,12 @@ cd ${MDA_DOCDIR} || die "Failed to 'cd ${MDA_DOCDIR}'. Run from the top level of
 mkdir ../${VERSION} && mv * ../${VERSION}
 
 git init
-git config user.name "${GIT_CI_USER}"
-git config user.email "${GIT_CI_EMAIL}"
+git config user.name github-actions
+git config user.email github-actions@github.com
```

Comment:
I can't find the gh actions details on this, do you have a reference for these? I keep finding "github-action@users.noreply.github.com", but that's in random github repos :/

Reply:
Do you think it matters? Didn't we use in the past "TravisCI@mdanlysis.org" – if anyone wants the commits associated with their name then they could put in theirs, e.g. lilyminium@users.noreply.github.com.

Reply:
I guess probably not? This whole area of using gh actions to push to repos just seems sparingly documented (unless I'm missing a really important page), particularly when it comes to examples. But I guess you're right, git doesn't really care if the email address is real.

Reply:
@IAlibay you're right I do see [the noreply](https://github.com/marketplace/actions/configure-git-credentials) credentials more often; I'm not sure where I stole the original from.